src/Psa/Output.purs

```diff
@@ -234,7 +234,7 @@ trimPosition lines pos =
       Just _            -> trimComment col l
       _                 -> Nothing
 
-  -- TODO: this breaks if "--" is insidentitye a quoted string.
```

Comment:
:smile:

Reply:
Some VIM vomit there probably.

FYI all this range trimming stuff is totally unnecessary now.

Reply:
Agreed. If there isn't a version support policy I'd propose my IDE one which is "current and previous major version". In any case removing this wouldn't completely break against older versions - it just gives looser ranges, right

Reply:
The ranges thing was fixed in 0.13, prior to that, yeah, the ranges included whitespace and comments.